target/linux/ar71xx/files/arch/mips/ath79/Makefile

```diff
@@ -55,6 +55,7 @@ obj-$(CONFIG_ATH79_MACH_AP147)			+= mach-ap147.o
 obj-$(CONFIG_ATH79_MACH_AP152)			+= mach-ap152.o
 obj-$(CONFIG_ATH79_MACH_AP90Q)			+= mach-ap90q.o
 obj-$(CONFIG_ATH79_MACH_AP96)			+= mach-ap96.o
+obj-$(CONFIG_ATH79_MACH_AP531B0)			+= mach-ap531b0.o
```

Comment:
Please, follow the general rule and keep all configs in alphabetical order. Also, keep proper column alignment.
You have also changed file attributes (`100644 → 100755`) which is **wrong** - please, fix it.